### PR TITLE
Add function to handle healthcheck

### PR DESCRIPTION
**Pull request by dora-korpar**
Creates healthcheck function called from S3.

### CODE REVIEW

**Comment by michael-zapata** [comment]
Wasn't the aim of the healthcheck to ease integration within the supervisor? Because sproxyd is already taken care of there.

**Comment by RachedBenMustapha** [comment]
@michael-zapata these are two totally different subjects

**Comment by michael-zapata** [comment]
Okay thanks, this is much clearer now.

**Comment by LaurenSpiegel** [comment]
@ironman-machine check it

File: lib/sproxyd.js [hunk 1]
```diff
@@ -295,6 +295,29 @@ class SproxydClient {
         const log = this.createLogger(reqUids);
         this._failover('DELETE', null, 0, key, 0, log, callback);
     }
+
+    /**
+    * This sends a GET request with healtcheck path to sproxyd
```
Comment (electrachong) [inline lib/sproxyd.js]: typo right? : healtcheck --> healthcheck

**Comment by LaurenSpiegel** [comment]
end to end passed: http://ci.ironmann.io/gh/scality/Integration/6146

**Pull request merged**